### PR TITLE
Add minetest.spawn_falling_node(pos)

### PR DESCRIPTION
**Pull request by zaoqi-unsafe**

### CODE REVIEW

**Comment by SmallJoker** [comment]
The code looks good. Please document this setting in the lua_api.txt file.

File: builtin/game/falling.lua [hunk 1]
```diff
@@ -119,9 +119,16 @@ local function spawn_falling_node(p, node)
 end
 
 function core.spawn_falling_node(pos)
+	if core.get_node(pos).name == "air" then
```
Comment (sfan5) [inline builtin/game/falling.lua]: what about `ignore`?

**Comment by bigfoot547** [comment]
Oh, and by the way, the test was successful. Am I the only one who translated the chinese in the commit message?

**Comment by paramat** [comment]
So what does this do? Change any node at the pos into a falling node?

File: builtin/game/falling.lua (hunk 1 above)
Comment (zaoqi-unsafe) [inline builtin/game/falling.lua]: What

**Comment by zaoqi-unsafe** [comment]
下接#5339

**Pull request closed without merging**